packages/foam-vscode/src/core/services/markdown-provider.ts

```diff
@@ -137,12 +137,22 @@ export function createMarkdownReferences(
         relativeUri = relativeUri.changeExtension('*', '');
       }
 
+      let label = link.rawText;
+      if (link.rawText.indexOf('[[') > -1) {
+        if (link.isEmbed) {
+          // ![[embedded-note]]
+          label = link.rawText.substring(3, link.rawText.length - 2);
+        } else {
+          // [[standard-note]]
+          label = link.rawText.substring(2, link.rawText.length - 2);
+        }
+      } else {
+        label = link.rawText;
```

Comment:
I believe this can be removed
```suggestion
```

Reply:
nice! updated, lemme know if it's too "busy"